### ISSUE TITLE
isort is removing the new line after __version__ = "0.0.0"

### ISSUE THREAD

**Issue by asheux** [timothycrosley/isort#1283]
- Before running isort
```py
__version__ = "0.58.1"

from starlette import status
...
```

- After running isort
```py
__version__ = "0.58.1"
from starlette import status
```

**Comment by jendrikseipp** [comment]
The same happens in 5.0.4 if there's an empty line between a comment and an import:

```
# My comment

import foo
import bar
``` 

But it seems you already fixed this in the development version @timothycrosley ?

**Comment by timothycrosley** [comment]
@jendrikseipp yes that's right :)

I appreciate the report, the fix is now out in isort 5.0.5 patch release with additional regression testing in place

Thanks!

~Timothy

**Comment by jendrikseipp** [comment]
Awesome, thanks!
